Tests/UnitTests/Networking/Responses/Fixtures/PaywallData-Sample1.json

```diff
@@ -37,13 +37,17 @@
                 "background": "#FF00AA",
                 "foreground": "#FF00AA22",
                 "call_to_action_background": "#FF00AACC",
-                "call_to_action_foreground": "#FF00AA"
+                "call_to_action_foreground": "#FF00AA",
+                "accent_1": "#FF0000",
+                "accent_2": "#00FF00"
```

Comment:
@guido732 @lburdock I named them `accent_1` and `accent_2` cause I think it's a bit more consistent. I also updated the doc.
Let me know what you think!